src/lib/skilling/skills/herblore/mixables/potions.ts

```diff
@@ -0,0 +1,457 @@
+		level: 53,
+		xp: 120,
+		inputItems: resolveNameBank({ 'Avantoe potion (unf)': 1, 'Kebbit teeth dust': 1 }),
+		tickRate: 2,
+		bankTimePerPotion: 0.3
+	},
+	{
+		name: 'Super strength (3)',
+		aliases: ['super strength (3)', 'super strength'],
+		id: itemID('Super strength (3)'),
+		level: 55,
+		xp: 125,
+		inputItems: resolveNameBank({ 'Kwuarm potion (unf)': 1, 'Limpwurt root': 1 }),
+		tickRate: 2,
+		bankTimePerPotion: 0.3
+	},
+	{
+		name: 'Weapon poison',
+		aliases: ['weapon poison'],
+		id: itemID('Weapon poison'),
+		level: 60,
+		xp: 137.5,
+		inputItems: resolveNameBank({ 'Kwuarm potion (unf)': 1, 'Dragon scale dust': 1 }),
+		tickRate: 2,
+		bankTimePerPotion: 0.3
+	},
+	{
+		name: 'Super restore (3)',
+		aliases: ['super restore (3)', 'super restore'],
+		id: itemID('Super restore (3)'),
+		level: 63,
+		xp: 142.5,
+		inputItems: resolveNameBank({
+			'Snapdragon potion (unf)': 1,
+			"Red spiders' eggs": 1
+		}),
+		tickRate: 2,
+		bankTimePerPotion: 0.3
+	},
+	{
+		name: 'Super defence (3)',
+		aliases: ['super defence (3)', 'super defence'],
+		id: itemID('Super defence (3)'),
+		level: 66,
+		xp: 150,
+		inputItems: resolveNameBank({ 'Cadantine potion (unf)': 1, 'White berries': 1 }),
+		tickRate: 2,
+		bankTimePerPotion: 0.3
+	},
+	{
+		name: 'Antidote+ (4)',
+		aliases: ['antidote+ (4)', 'antidote+'],
+		id: itemID('Antidote+ (4)'),
+		level: 68,
+		xp: 155,
+		inputItems: resolveNameBank({ 'Coconut milk': 1, Toadflax: 1, 'Yew roots': 1 }),
+		tickRate: 2,
+		bankTimePerPotion: 0.45
+	},
+	{
+		name: 'Antifire potion (3)',
+		aliases: ['antifire potion (3)', 'antifire potion', 'antifire'],
+		id: itemID('Antifire potion (3)'),
+		level: 69,
+		xp: 157.5,
+		inputItems: resolveNameBank({
+			'Lantadyme potion (unf)': 1,
+			'Dragon scale dust': 1
+		}),
+		tickRate: 2,
+		bankTimePerPotion: 0.3
+	},
+	{
+		name: 'Ranging potion (3)',
+		aliases: ['ranging potion (3)', 'ranging potion', 'ranging'],
+		id: itemID('Ranging potion (3)'),
+		level: 72,
+		xp: 162.5,
+		inputItems: resolveNameBank({
+			'Dwarf weed potion (unf)': 1,
+			'Wine of zamorak': 1
+		}),
+		tickRate: 2,
+		bankTimePerPotion: 0.3
+	},
+	{
+		name: 'Weapon poison(+)',
+		aliases: ['weapon poison(+)'],
+		id: itemID('Weapon poison(+)'),
+		level: 73,
+		xp: 165,
+		inputItems: resolveNameBank({
+			'Coconut milk': 1,
+			'Cactus spine': 1,
+			"Red spiders' eggs": 1
+		}),
+		tickRate: 2,
+		bankTimePerPotion: 0.3
+	},
+	{
+		name: 'Magic potion (3)',
+		aliases: ['magic potion (3)', 'magic potion', 'magic'],
+		id: itemID('Magic potion (3)'),
+		level: 76,
+		xp: 172.5,
+		inputItems: resolveNameBank({ 'Lantadyme potion (unf)': 1, 'Potato cactus': 1 }),
+		tickRate: 2,
+		bankTimePerPotion: 0.3
+	},
+	{
+		name: 'Stamina potion (4)',
+		aliases: ['stamina potion (4)', 'stamina potion', 'stamina'],
+		id: itemID('Stamina potion (4)'),
+		level: 77,
+		xp: 102,
+		inputItems: resolveNameBank({ 'Super energy (4)': 1, 'Amylase crystal': 4 }),
+		tickRate: 2,
```

Comment:
slightly increase tickRate or banking time to make the xp/hr reflect the wiki better since all other potions seems to reflect the wiki page.

Reply:
fixed